### ISSUE TITLE
DefaultApp.tsx missing in build

### ISSUE THREAD

**Issue by tomaszrondio** [toomuchdesign/next-page-tester#103]
## 🐛 Bug Report

Cannot find module 'node_modules/next-page-tester/dist/_app/DefaultApp.tsx' from 'node_modules/next-page-tester/dist/loadPage.js'

## To Reproduce

install from npm and run example test

## Run `npx envinfo --preset jest`

Paste the results here:

```bash
System:
    OS: macOS 11.1
    CPU: (8) x64 Apple M1
  Binaries:
    Node: 14.15.2 - /usr/local/bin/node
    Yarn: 1.22.10 - ~/Desktop/projekty/publisher_pwa/node_modules/.bin/yarn
    npm: 6.14.9 - /usr/local/bin/npm
  npmPackages:
    jest: 26.6.3 => 26.6.3
```

**Comment by Meemaw** [comment]
Hi @tomaszrondio 👋 Could you also share `NextJS` version you are using?

**Comment by tomaszrondio** [comment]
that is easy :) "next": "10.0.3". I just switched to 10.0.5 and same happens

**Comment by tomaszrondio** [comment]
<img width="210" alt="Screenshot 2021-01-14 at 13 00 20" src="https://user-images.githubusercontent.com/4939724/104588339-81d8f900-5668-11eb-987e-594f9007d89f.png">

**Comment by toomuchdesign** [comment]
Nice one! Since we don't import `DefaultApp.tsx` (but manually require it), Typescript won't include it in its dist folder.
Thanks @tomaszrondio

We should add tests targeting the bundled code.

**Comment by tomaszrondio** [comment]
My pleasure. Always happy to break stuff.

**Comment by Meemaw** [comment]
I believe temporary fix would be to create a [custom app file](https://nextjs.org/docs/advanced-features/custom-app) until this is fixed.

**Comment by tomaszrondio** [comment]
I do have custom `_app.jsx ` with lots of stuff

**Comment by toomuchdesign** [comment]
I would have expected this issue not to happen when a custom `_app` file is available. @tomaszrondio are you maybe testing with `useApp` option set to `false`?

BTW, I'll publish a patch for the failing require today.

**Comment by toomuchdesign** [comment]
Published a fix with version 0.16.1 🚢 . Thanks for reporting @tomaszrondio!

Would mind letting us know if the issue is solved?

**Comment by tomaszrondio** [comment]
I updated to 0.16.1 and it works!
